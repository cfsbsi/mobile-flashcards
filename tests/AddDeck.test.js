import React from 'react';
import renderer from 'react-test-renderer';
import AddDeck from "../components/AddDeck";

it('renders correctly', () => {
    const tree = renderer.create(<AddDeck />).toJSON();
    expect(tree).toMatchSnapshot();
});