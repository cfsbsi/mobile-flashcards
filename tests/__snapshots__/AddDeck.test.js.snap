// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
      },
      undefined,
    ]
  }
>
  <View
    style={
      Array [
        Object {
          "marginBottom": 20,
          "marginTop": 70,
        },
        undefined,
      ]
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "fontSize": 30,
            "fontWeight": "bold",
            "textAlign": "center",
          },
          undefined,
        ]
      }
    >
      What is the title
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "fontSize": 30,
            "fontWeight": "bold",
            "textAlign": "center",
          },
          undefined,
        ]
      }
    >
      of your new
    </Text>
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "fontSize": 30,
            "fontWeight": "bold",
            "textAlign": "center",
          },
          undefined,
        ]
      }
    >
      deck?
    </Text>
  </View>
  <TextInput
    allowFontScaling={true}
    onChangeText={[Function]}
    style={
      Array [
        Object {
          "borderColor": "gray",
          "borderRadius": 7,
          "borderWidth": 2,
          "height": 50,
          "marginTop": 40,
          "width": 300,
        },
        undefined,
      ]
    }
    value=""
  />
  <View
    accessibilityComponentType={undefined}
    accessibilityLabel={undefined}
    accessibilityTraits={undefined}
    accessible={true}
    collapsable={undefined}
    hasTVPreferredFocus={undefined}
    hitSlop={undefined}
    isTVSelectable={true}
    nativeID={undefined}
    onLayout={undefined}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "backgroundColor": "black",
        "borderRadius": 7,
        "marginTop": 80,
        "opacity": 1,
        "paddingBottom": 10,
        "paddingLeft": 15,
        "paddingRight": 15,
        "paddingTop": 10,
      }
    }
    testID={undefined}
    tvParallaxProperties={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
      style={
        Array [
          Object {
            "color": "white",
            "fontSize": 30,
            "textAlign": "center",
          },
          undefined,
        ]
      }
    >
      SUBMIT
    </Text>
  </View>
</View>
`;
